launch-script/src/proposals/create_treasury_pool.rs

```diff
@@ -0,0 +1,75 @@
+// =========================================================================
+// Create collateral pool accounts
+// =========================================================================
+
+use crate::prelude::*;
+
+const TREASURY_POOL_ACCOUNT_COUNT: u32 = 10;
```

Comment:
Get the actual count of treasury pool accounts from evm_loader.

Reply:
Let's fix it during parse_els procedure refactoring